src/index.html

```diff
@@ -3,10 +3,404 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#home" class="link">
+        <img
+          src="images/logo.png"
+          alt="Logo"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptopsandcomputers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="container">
+        <div class="card container__card" data-qa="card">
+          <div class="card__container">
+            <img
+              src="./images/imac.jpeg"
+              alt="Product image"
+              class="card__image"
+            >
+
+            <h1 class="card__caption">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
+
+            <h2 class="card__code">Product code: 195434</h2>
+
+            <div class="card__rating">
+              <div class="stars card__stars card__stars--4">
```

Comment:
```suggestion
              <div class="stars card__stars stars--4">
```
Modifier should be added to stars block, not card element